apps/web/components/integrations/CalendarListContainer.tsx

```diff
@@ -207,7 +207,11 @@ function ConnectedCalendarsList(props: Props) {
   );
 }
 
-export function CalendarListContainer(props: { heading?: boolean; fromOnboarding?: boolean }) {
+export function CalendarListContainer(props: {
+  heading?: boolean;
+  items: inferQueryOutput<"viewer.integrations">["items"];
```

Comment:
Passing down already fetched items to just show them, instead of having to fetch the passed down variant